src/internal/connector/onedrive/collection.go

```diff
@@ -198,11 +205,16 @@ func (oc *Collection) populateItems(ctx context.Context) {
 		m.Unlock()
 	}
 
-	for _, item := range oc.driveItems {
+	for id, item := range oc.driveItems {
 		if oc.ctrl.FailFast && errs != nil {
 			break
 		}
 
+		if item == nil {
+			errUpdater(id, errors.New("nil item"))
+			continue
+		}
```

Comment:
Is this expected?

Reply:
Not particularly, but I had a test that paniced on it.  Figured we're better off protecting ourselves against a panic than risking it.